packages/kit/src/types/internal.d.ts

```diff
@@ -566,5 +567,11 @@ export type RemoteInfo =
 			inputs?: RemotePrerenderInputsGenerator;
 	  };
 
+export type RecordSpan = <T>(options: {
```

Comment:
This has to be its own type so that we can import it in `internal` -- it doesn't like it if we import the type of the function from `runtime`